src/main/java/org/osiam/resources/scim/Resource.java

```diff
@@ -103,8 +102,9 @@ public Meta getMeta() {
      *
      * @return a the list of schemas as a {@link Set}
      */
-    public Set<String> getSchemas() {
-        return ImmutableSet.copyOf(schemas);
+    @NotEmpty
+    public Set<String> getSchemas(){
+        return schemas;
```

Comment:
Why this? I mean, it's the SCIM stuff of the server now, so we probably don't need the immutability here, but I still wonder, whether it will break #143.